### PR TITLE
add lua support for  `plural test`

### PR DESCRIPTION
**Pull request by zreigz**
## Summary
add a Lua support for `plural test` command

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: cmd/plural/template.go [hunk 1]
```diff
@@ -30,35 +33,67 @@ func testTemplate(c *cli.Context) error {
 			continue
 		}
 
-		ctx := installation.Context
-		tmpl, err := template.MakeTemplate(string(testTemplate))
+		var errList []error
+		vals := genDefaultValues(conf, installation)
+
+		output, err := luaTmpValues(string(testTemplate), vals)
```
Comment (michaeljguarino) [inline cmd/plural/template.go:39]: Perhaps we should have a flag that tells what template type it is?

Comment (michaeljguarino) [reply cmd/plural/template.go:39]: brute forcing isn't ideal and we definitely will know server-side

Comment (zreigz) [reply cmd/plural/template.go:39]: I will add flag for the type

Comment (zreigz) [reply cmd/plural/template.go]: done

**Comment by zreigz** [comment]
@michaeljguarino the flag was added. PTAL

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**